packages/eslint-plugin-template/tests/configs.test.ts

```diff
@@ -1,15 +1,18 @@
 import eslintPluginTemplate from '../src';
 
+const ESLINT_PLUGIN_PREFFIX = '@angular-eslint/template/';
+
 interface Config {
   extends?: string | string[];
   rules?: { [ruleName: string]: string | object };
   overrides?: Config[];
 }
 
 function containsRule(config: any, ruleName: string): boolean {
```

Comment:
I know this is something that you didn't changed, but would you mind to get rid of `any` (here and in other `configs.test`)?

```suggestion
function containsRule(config: Config, ruleName: string): boolean {
```

Note that in doing so you may need to wrap all the return section with `Boolean` (not only the first conditional) and also you can get rid of `config: Config` as it would be inferred automatically.